lib/limits.c

```diff
@@ -118,6 +118,10 @@ static int check_logins (const char *name, const char *maxlogins)
 	unsigned long limit, count;
 
 	if (str2ul(&limit, maxlogins) == -1) {
+		if (errno == ERANGE) {
+			SYSLOG((LOG_WARN, "Invalid maxlogins value\n"));
+			return LOGIN_ERROR_LOGIN;
+		}
 		return 0;
 	}
```

Comment:
@hallyn Please check this part carefully.  Look at <https://github.com/shadow-maint/shadow/commit/10396f953613f9ecd1aacd30ba457a32f583184a#r136637039>